src/ApiCheck/ApiListing/MemberKind.cs

```diff
@@ -0,0 +1,16 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using Newtonsoft.Json;
+using Newtonsoft.Json.Converters;
+
+namespace ApiCheck.Description
+{
+    [JsonConverter(typeof(StringEnumConverter))]
+    public enum MemberKind
+    {
+        Constructor,
+        Method,
```

Comment:
Might be good to add a doc comment to this enum value specifically to indicate that this represents the accessor methods of properties and events, and thus there are no specific member kinds for properties and events.